extendeddatacrossword_test.go

```diff
@@ -344,3 +345,143 @@ func createTestEds(codec Codec, shareSize int) *ExtendedDataSquare {
 
 	return eds
 }
+
+func TestCorruptedEdsReturnsErrByzantineData_UorderedShares(t *testing.T) {
+	shareSize := 64
+	namespaceSize := 1
+	one := bytes.Repeat([]byte{1}, shareSize)
+	two := bytes.Repeat([]byte{2}, shareSize)
+	three := bytes.Repeat([]byte{3}, shareSize)
+	sharesValue := []int{1, 2, 3, 4}
+	tests := []struct {
+		name          string
+		coords        [][]uint
+		values        [][]byte
+		wantErr       bool
+		corruptedAxis Axis
+		corruptedInd  uint
```

Comment:
[nit] it wasn't immediately obvious what `Ind` meant but it appears short for index. This suggestion can't be applied directly b/c it implies renaming the variable in this scope

```suggestion
		corruptedIndex  uint
```

Reply:
> This suggestion can't be applied directly b/c it implies renaming the variable in this scope

Can you please expand on this second part of your comment? Nevertheless, I applied the first part of your feedback in https://github.com/celestiaorg/rsmt2d/pull/242/commits/1a4e055a9427c3e37e28dc72d22922d4dc754443